api/prometheus/v1/api.go

```diff
@@ -675,16 +675,10 @@ func (h *httpAPI) QueryRange(ctx context.Context, query string, r Range) (model.
 	u := h.client.URL(epQueryRange, nil)
 	q := u.Query()
 
-	var (
-		start = r.Start.Format(time.RFC3339Nano)
-		end   = r.End.Format(time.RFC3339Nano)
-		step  = strconv.FormatFloat(r.Step.Seconds(), 'f', 3, 64)
-	)
-
 	q.Set("query", query)
-	q.Set("start", start)
-	q.Set("end", end)
-	q.Set("step", step)
+	q.Set("start", formatTime(r.Start))
+	q.Set("end", formatTime(r.End))
+	q.Set("step", strconv.FormatFloat(r.Step.Seconds(), 'f', 3, 64))
```

Comment:
Not really related to this change, but I'm wondering if there was a reason to use `3` here instead of `-1`. We do the same for the timestamp formatting now, so perhaps adjust it to `-1`, now that we are on it?

Reply:
No reason, it was just that already (I just changed the format to match the other methods). Updated :)